hmy/api_backend.go

```diff
@@ -370,7 +370,8 @@ func (b *APIBackend) GetValidatorInformation(
 	computed := availability.ComputeCurrentSigning(
 		snapshot, wrapper,
 	)
-	computed.BlocksLeftInEpoch = shard.Schedule.BlocksPerEpoch() - computed.ToSign.Uint64()
+	beaconChainBlocks := uint64(b.hmy.BeaconChain().CurrentBlock().Header().Number().Int64()) % shard.Schedule.BlocksPerEpoch()
```

Comment:
why not just get the blockchain current block header?

Reply:
Want to compute blocks left on Beaconchain until next epoch.